src/tests/build.cmd

```diff
@@ -174,7 +227,7 @@ REM Set up the directory for MSBuild debug logs.
 set MSBUILDDEBUGPATH=%__MsbuildDebugLogsDir%
 
 set __CommonMSBuildArgs="/p:TargetOS=%__TargetOS%"
-set __CommonMSBuildArgs=%__CommonMSBuildArgs% "/p:Configuration=%__BuildType%"
+set __CommonMSBuildArgs=%__CommonMSBuildArgs% "/p:Configuration=%__BuildType%" "/p:LibrariesConfiguration=%__BuildType%"
```

Comment:
The instructions for using corerun led me to getting tripped up here, because I was passing "debug" as an argument, but the `LibrariesConfiguration` property was not getting set and therefore it was still defaulting to `Release`. That would require `/p:LibrariesConfiguration=Debug` to also be specified in order for the instructions to work. Instead, this now makes the configuration argument apply to both properties needed in the `.proj`.

Reply:
It is intentional that the runtime tests default to release build of the libraries. This was discussed in https://github.com/dotnet/runtime/pull/75033

Reply:
OK; thanks! I'll augment the doc to reflect that detail and add the instruction for specifying the `/p:LibrariesConfiguration=Debug` argument.